third_party/ucd/emoji-data.txt

```diff
@@ -0,0 +1,1315 @@
+# Emoji Data for UTR #51
```

Comment:
You need to update the README.third_party file to point to location and version of this file.